tardis/utilities/utils.py

```diff
@@ -42,3 +42,47 @@ def htcondor_csv_parser(htcondor_input, fieldnames, delimiter="\t", replacements
                 key: value if value not in replacements.keys() else replacements[value]
                 for key, value in row.items()
             }
+
+
+def slurm_cmd_option_formatter(options):
+    """
+    Formats name/value pais in the `options` dict as `--<name> value` suitable
+    for passing to SLURM command line tools.
+
+    :param options:  name/value pairs
+    :type options: AttributeDict
+    """
+    options = (
+        f"--{name} {value}" if value is not None else f"--{name}"
+        for name, value in options.items()
+    )
+
+    return " ".join(options)
+
+
+def slurm_csv_parser(slurm_input, fieldnames, delimiter=" ", replacements=None):
```

Comment:
What is the difference to the already exisiting `htcondor_csv_parser`? Can we merge them?

Reply:
merged